website/src/css/pages/_events.scss

```
@@ -1,5 +1,5 @@
-.events {
+.event {
     &__item h6 {
-	color: #ccc;
+        color: rgb(105, 120, 130);
```

```suggestion
        color: $light-2;
```

You can get a similar color using $light-2 from our color palette, so we have consistent colors throughout our site (see _colors.scss). Does this color work?